### PR TITLE
Update limits-specifications-teams.md

### PR DESCRIPTION
**Pull request by Reeced40**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2993
Added information.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/limits-specifications-teams.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ This article describes some of the limits, specifications, and other requirement
 |Feature    | Maximum limit |
 |-----------|---------------|
 |Number of teams a user can create | Subject to a 250 object limit&sup1;         |
+|Number of Groups a user can be a member of|1,000|
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md]:
```suggestion
|Number of groups a user can be a member of|1,000|
```

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LolaJacobsen** [comment]
@karvell @phecdalouie Please review this PR.

**Comment by LolaJacobsen** [comment]
OK per PM @Phecdalouie.

**Pull request merged**